libbeat/common/transport/tlscommon/types.go

```diff
@@ -70,6 +70,12 @@ var tlsCurveTypes = map[string]tlsCurveType{
 	"P-521": tlsCurveType(tls.CurveP521),
 }
 
+var tlsRenegotiationSupportTypesInverse = map[tlsRenegotiationSupport]string{
+	tlsRenegotiationSupport(tls.RenegotiateNever):          "never",
+	tlsRenegotiationSupport(tls.RenegotiateOnceAsClient):   "once",
+	tlsRenegotiationSupport(tls.RenegotiateFreelyAsClient): "freely",
+}
```

Comment:
See line 61. For cipher suites we tend to build the inverse dynamically on startup, so to reduce the coupling between 2 structures.